src/components/30-organisms/container/story.js

```diff
@@ -22,9 +22,7 @@ export const Container = () => {
 
   const childsText = text('text', 'Some children');
 
-  const template = html`
-    <axa-container>${childsText}</axa-container>
-  `;
+  const template = html` <axa-container>${childsText}</axa-container> `;
```

Comment:
How come this transformation is safe if the newlines that were there before are deleted????